src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -137,4 +137,117 @@ export default class GroupComparisonStore {
             }
         }
     });
+
+    public readonly sampleSet = remoteData({
+        await: () => [
+            this.samples
+        ],
+        invoke: () => {
+            return Promise.resolve(_.keyBy(this.samples.result!, sample => sample.studyId+sample.sampleId));
+        }
+    });
+
+    public readonly patientToAnalysisGroups = remoteData({
+        await: () => [
+            this.sampleGroups,
+            this.sampleSet
+        ],
+        invoke: () => {
+            let sampleSet = this.sampleSet.result!
+            return Promise.resolve(_.reduce(this.sampleGroups.result, (acc, next) => {
+                next.sampleIdentifiers.forEach(sampleIdentifier => {
+                    let sample = sampleSet[sampleIdentifier.studyId+sampleIdentifier.sampleId]
+                    if(sample){
+                        let groups = acc[sample.uniquePatientKey] || [];
+                        groups.push(next.id);
+                        acc[sample.uniquePatientKey] = groups;
+                    }
+                })
+                return acc;
+            }, {} as { [id: string]: string []}));
+        }
+    });
+
+    readonly survivalClinicalDataExists = remoteData<boolean>({
+        await:()=>[
+            this.studyIds,
+            this.samples
+        ],
+        invoke:async()=>{
+            const filter: ClinicalDataMultiStudyFilter = {
+                attributeIds: ["OS_STATUS", "OS_MONTHS", "DFS_STATUS", "DFS_MONTHS"],
```

Comment:
we should make sure these attribute ids are not duplicated in resultsViewPageStore (put them in a some config file)